Uppsala/UppEt41.xml

```diff
@@ -0,0 +1,501 @@
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="190r" to="197v"></locus>
+                                        </item>
+                                        
+                                        <item xml:id="q26" n="25">
+                                            <num value="5">፭</num>
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="198r" to="205v"></locus>
+                                        </item>
+                                        
+                                        <item xml:id="q27" n="26">
+                                            <num value="6">፮</num>
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="206r" to="213v"></locus>
+                                        </item>
+                                        
+                                        <item xml:id="q28" n="27">
+                                            <num value="7">፯</num>
+                                            <dim unit="leaf">6</dim>
+                                            <locus from="214r" to="219v"></locus>
+                                        </item>
+                                        
+                                    </list>
+                                </collation>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="3" writtenLines="31 34">
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc><bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">130</citedRange>
+                                </bibl> says that the manuscript is written by a "beautiful and even hand" 
+                                    (<foreign xml:lang="de">von schöner und gleichmäßiger Hand</foreign>).</desc>
+                                <date when="1605">1605.</date>
+                            </handNote>
+                        </handDesc>
+                        
+                        <decoDesc>
+                            
+                            <decoNote xml:id="d1" type="doodles">
+                                <locus target="1r"/>
+                                <desc>Doodles of an unarmed man on a horse and two men with hoods, identified by
+                                    <bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">129</citedRange>
+                                    </bibl> as priests. The man on a horse has two captions by different hands:</desc>
```

Comment:
If you mention the horse for the second time in a note, you should refer to it with "the horse".

Reply:
Not sure in this context, but I reformulated it